forge-gui/res/cardsfolder/upcoming/writhing_necromass.txt

```diff
@@ -0,0 +1,9 @@
+Name:Writhing Necromass
+ManaCost:6 B
+Types:Creature Zombie Giant
+PT:5/5
+K:Deathtouch
+S:Mode$ ReduceCost | ValidCard$ Card.Self | Type$ Spell | Amount$ X | EffectZone$ All | Description$ This spell costs {1} less to cast for each creature card in your graveyard.
+SVar:X:Count$TypeInYourYard.Creature
+DeckHas:Ability$Graveyard
```

Comment:
This card doesn't have any ability that would directly interact with the graveyard (put something there etc.), please check the other cards that have Ability$Graveyard for reference